packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -54,8 +55,8 @@ const MT5AccountCard = ({
                 {icon && <IconComponent />}
                 <div className='mt5-account-card__type--description'>
                     <h1 className='mt5-account-card--heading'>{title}</h1>
-                    {!existing_data && <p className='mt5-account-card--paragraph'>{descriptor}</p>}
-                    {existing_data && existing_data.display_balance && (
+                    {(!existing_data || !is_logged_in) && <p className='mt5-account-card--paragraph'>{descriptor}</p>}
+                    {existing_data && existing_data.display_balance && is_logged_in && (
```

Comment:
can existing_data exist without is_logged_in?

Reply:
and could `has_mt5_account` be used instead of `is_logged_in` perhaps?

Reply:
existing_data seems to exist even when users have logged out...and has_mt5_account can't be used cause the user could have a demo account and we don't want to display the paragraph when there is data for the demo account